src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -141,6 +141,8 @@ g.test('color_formats_must_be_renderable')
 
     const descriptor = t.getDescriptor({ colorStates: [{ format }] });
 
+    await t.selectDeviceOrSkipTestCase(info.extension);
```

Comment:
nit: Do this as early as possible (right after `info`)

Reply:
Ack. Done.